### PR TITLE
feat: init dev portal Webview

### PR DESCRIPTION
**Pull request by chengoramazon**
## Problem

UX refinement for Amazon Q `/dev`

## Solution

- Transit UX to a dedicated Webview

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/core/src/amazonqFeatureDev/session/session.ts [hunk 1]
```diff
@@ -84,6 +91,8 @@ export class Session {
             span.record({ amazonqConversationId: this._conversationId, credentialStartUrl: AuthUtil.instance.startUrl })
         })
 
+        this.showDevPortabl(msg)
```
Comment (willyyhuang) [inline packages/core/src/amazonqFeatureDev/session/session.ts:94]: typo? `showDevPortal`

File: packages/core/src/amazonqFeatureDev/webview/devPortalWebview.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { VueWebview } from '../../webviews/main'
+
+class DevPortalWebview extends VueWebview {
+    public static readonly sourcePath: string = 'src/amazonqFeatureDev/webview/vue/index.js'
```
Comment (willyyhuang) [inline packages/core/src/amazonqFeatureDev/webview/devPortalWebview.ts:4]: `index.js` or `ts`

Comment (chengoramazon) [reply packages/core/src/amazonqFeatureDev/webview/devPortalWebview.ts:4]: compiled js

File: packages/core/src/amazonqFeatureDev/session/session.ts [hunk 3]
```diff
@@ -84,6 +91,8 @@ export class Session {
             span.record({ amazonqConversationId: this._conversationId, credentialStartUrl: AuthUtil.instance.startUrl })
         })
 
+        this.showDevPortal(msg)
```
Comment (willyyhuang) [inline packages/core/src/amazonqFeatureDev/session/session.ts:94]: should this be a static title?

Comment (chengoramazon) [reply packages/core/src/amazonqFeatureDev/session/session.ts:94]: No, how users distinguish different task

File: packages/core/src/amazonqFeatureDev/webview/devPortalWebview.ts
Comment (justinmk3) [inline packages/core/src/amazonqFeatureDev/webview/devPortalWebview.ts:1]: no need to repeat "webview" in the name, it lives in a webview/ folder already. Please follow the conventions of (most of) the existing webviews.

    packages/core/src/amazonqFeatureDev/webview/devPortal.ts

**Review by willyyhuang — APPROVED** [review]

File: packages/core/src/amazonqFeatureDev/session/session.ts [hunk 4]
```diff
@@ -84,6 +91,8 @@ export class Session {
             span.record({ amazonqConversationId: this._conversationId, credentialStartUrl: AuthUtil.instance.startUrl })
         })
 
+        await this.showDevPortal(msg)
```
Comment (chengoramazon) [inline packages/core/src/amazonqFeatureDev/session/session.ts:94]: This PR will be discarded.

```suggestion
        ## No longer needed
        await this.showDevPortal(msg)
```

**Pull request closed without merging**